specificationRepositoryConfiguration.json

```diff
@@ -8,8 +8,9 @@
       "configFilePath": "eng/swagger_to_sdk_config.json"
     },
     "azure-sdk-for-java": {
-      "integrationRepository": "azure-sdk/azure-sdk-for-java",
-      "mainRepository": "Azure/azure-sdk-for-java",
+      "integrationRepository": "test-repo-billy/azure-sdk-for-java",
+      "mainRepository": "test-repo-billy/azure-sdk-for-java",
+      "mainBranch": "haoling-test",
```

Comment:
Seems this branch name isn't the one you created in this PR.

Reply:
it's azure-sdk-for-java's branch.

Reply:
I checked the log, seems it is not using test billy project.